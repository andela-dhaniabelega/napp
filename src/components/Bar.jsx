import React from 'react';
import '../../public/sass/styles.scss'; 

/**
 * Stateless component
 * Simply renders horizontal bar above sliders
 *
 * @return {void}
 */
const Bar = () => (
  <div className="main_menu_area">
    <ul id="nav">
      <span className="u-id" />         
    </ul>
  </div>
);

export default Bar;
