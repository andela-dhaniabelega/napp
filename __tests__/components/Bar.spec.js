import React from 'react';
import renderer from 'react-test-renderer'
import Bar from '../../src/components/Bar';


describe('Bar component', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<Bar />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});