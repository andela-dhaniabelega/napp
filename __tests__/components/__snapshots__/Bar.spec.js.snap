// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Us Page renders a snapshot 1`] = `
<div
  className="main_menu_area"
>
  <ul
    id="nav"
  >
    <span
      className="u-id"
    />
  </ul>
</div>
`;
